release/rebuild_bom_files.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+source _bom.sh
+source _git.sh
+source _liferay_common.sh
+source _package.sh
+source _product.sh
+source _promotion.sh
+
+function check_usage {
+	if [ -z "${LIFERAY_RELEASE_PRODUCT_NAME}" ] ||
+	   [ -z "${LIFERAY_RELEASE_VERSION}" ]
+	then
+		print_help
+	fi
+
+	_BUILD_TIMESTAMP=$(date +%s)
+
+	_PRODUCT_VERSION="${LIFERAY_RELEASE_VERSION}"
+
+	_ARTIFACT_RC_VERSION="${_PRODUCT_VERSION}-${_BUILD_TIMESTAMP}"
+
+	_RELEASE_TOOL_DIR=$(dirname "$(readlink /proc/$$/fd/255 2>/dev/null)")
+
+	lc_cd "${_RELEASE_TOOL_DIR}"
```

Comment:
@lucasmiranda0, can you please try to save the declaration of this variable?

Please make sure there is no other cases like this